tests/html/script-test.html

```diff
@@ -33,8 +33,9 @@
         })(method);
       }
 
-      rudderanalytics.load("1cA1w20Fe34TdZrtGshutNY8Fm1", "http://localhost:8080", {
```

Comment:
include this test file inside the moengage dest folder.
also remove so many commented lines and use the final tests that were run.

Reply:
I created a test folder with different cases which I finally ran including the results.